src/gtk/widgets/recent_info.rs

```diff
@@ -141,23 +142,22 @@ impl RecentInfo {
 
     pub fn has_group(&self, group_name: &str) -> bool {
         unsafe {
-            to_bool(
-                ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
-                                               group_name.borrow_to_glib().0))
+            to_bool(ffi::gtk_recent_info_has_group(GTK_RECENT_INFO(self.unwrap_widget()),
+                group_name.borrow_to_glib().0))
         }
```

Comment:
I am not even sure when you ever want to know what function that argument will be passed to, unless you are editing those functions or doing some kind of code review, in which case the difference can clearly be neglected as it is hard to measure the difference in time. In both cases you will be moving the cursor often, and for most editors it should require minimal effort.

What I do care more about is the amount of vertical space that is used, especially when it is the result of minimising the amount of horizontal space that can be used due to alignment constraints imposed by these decorative purposes I am talking about. However, I do agree that these are hard to notice in these particular cases, but in such cases you should consider functions that accept more than one argument, where you usually end up allocating a single line per argument, as well as the overall effect. An example, which you have spotted already, can be found at `ffi::gtk_recent_info_get_application_info` (line 92).